types/shopify-buy/index.d.ts

```diff
@@ -377,12 +383,17 @@ declare namespace ShopifyBuy {
         /**
          * ID of line item variant.
          */
-        variant_id: string | number;
+        variantId: string | number;
 
         /**
          * Title of variant.
          */
-        variant_title: string;
+        variantTitle: string;
+    }
+
+    export interface LineItemAdded {
```

Comment:
Perhaps a better name is `NewLineItem` or `LineItemToAdd`.  My issue is with the past tense `Added`, which is not accurate since _it will be_ added, but hasn't been added yet.